docs/src/main/asciidoc/index.adoc

```diff
@@ -15,9 +15,9 @@ Cloud Platform (GCP).
 Currently, Spring Cloud GCP lets you leverage the power and simplicity of the Spring framework to:
 
 1. Publish and subscribe from Google Cloud Pub/Sub topics
-2. Exchange messages with Spring Integration using Google Cloud Pub/Sub on the background
+2. Configure Spring JDBC with a few properties to use Google Cloud SQL
```

Comment:
JDBC and Spring Data?

Reply:
Nothing specific in our integration for Spring Data, so I'd rather omit it here.

Reply:
ok